src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -36,25 +41,70 @@ class ContainerConfigurator extends AbstractConfigurator
     private $file;
     private $anonymousCount = 0;
     private $env;
+    private $buildDir;
+    private $generator;
 
-    public function __construct(ContainerBuilder $container, PhpFileLoader $loader, array &$instanceof, string $path, string $file, string $env = null)
+    public function __construct(ContainerBuilder $container, PhpFileLoader $loader, array &$instanceof, string $path, string $file, string $env = null, ?string $buildDir = null, GeneratorInterface $generator = null)
     {
         $this->container = $container;
         $this->loader = $loader;
         $this->instanceof = &$instanceof;
         $this->path = $path;
         $this->file = $file;
         $this->env = $env;
+        $this->buildDir = $buildDir;
+        $this->generator = $generator;
     }
 
     final public function extension(string $namespace, array $config)
+    {
+        $this->throwIfExtensionDoesNotExist($namespace);
+
+        $this->container->loadFromExtension($namespace, static::processValue($config));
+    }
+
+    /**
+     * @param string $namespace example extension alias ("framework") or FQCN string
+     *                          for a class implementing ConfigBuilderInterface
+     */
+    final public function configBuilder(string $namespace): ConfigBuilderInterface
```

Comment:
do we need this method? what about allowing only passing via arguments as resolved by the loader?
using this method doesn't work this autocompletion anyway

Reply:
You are correct. Thank you.